beluga/models.py

```diff
@@ -78,9 +78,18 @@ class Event(Base):
 
     start_time = sa.Column(sa.types.DateTime())
     end_time = sa.Column(sa.types.DateTime())
+    start_time_local = sa.Column(sa.types.DateTime())
+    end_time_loal = sa.Column(sa.types.DateTime())
+    timezone = sa.Column(sa.String(50))
     location = sa.Column(sa.types.JSON)
     title = sa.Column(sa.String(200))
     attendees = sa.Column(sa.types.JSON)
+    capacity = sa.Column(sa.Integer())
+    logo = sa.Column(sa.types.JSON)
```

Comment:
Logo is... JSON? 🤔

Reply:
Logo is a little JSON bundle of URL's for different logo sizes. Unless you just want me to pick a size?